src/Agent/NewRelic/Agent/Core/Aggregators/LogEventAggregator.cs

```diff
@@ -41,7 +41,7 @@ public LogEventAggregator(IDataTransportService dataTransportService, IScheduler
         }
 
         protected override TimeSpan HarvestCycle => _configuration.LogEventsHarvestCycle;
-        protected override bool IsEnabled => _configuration.LogEventCollectorEnabled;
+        protected override bool IsEnabled => _configuration.LogEventCollectorEnabled && _configuration.LogEventsMaxSamplesStored > 0;
```

Comment:
I thind DefaultConfiguration.LogEventCollectorEnabled already includes this in it's logic